app/src/main/res/color/material_button_text.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The name of this file seems very generic. I don't know if that's expected or nor, but it doesn't give much info about when to use it.

Reply:
Changing it to be broken site specific.